packages/lib/hooks/useApp.ts

```diff
@@ -1,8 +1,8 @@
 import { useSession } from "next-auth/react";
 
-import { trpc } from "@calcom/trpc/react";
+import { trpc, RouterOutputs } from "@calcom/trpc/react";
 
-export default function useApp(appId: string) {
+export default function useApp(appId: string): RouterOutputs["viewer"]["appById"][number] {
```

Comment:
Needed to add the output type or else I was getting the `any` type error

Reply:
This is not a solution, the real solution is figuring out with the type inference is lost: 

<img width="524" alt="image" src="https://user-images.githubusercontent.com/3504472/211076420-49a19857-0f5b-44ff-8bc1-73011fe2427e.png">

My guess is that `trpc` is not a dependency in `packages/lib`